public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -781,6 +781,22 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
     return this.serializer.getSaveModel(this);
   }
 
+  // Get the dashboard in native K8s form (using the appropriate apiVersion)
+  getSaveResource(options: SaveDashboardAsOptions): ResourceForCreate<unknown> {
```

Comment:
@dprokop -- this is a basic stub that hardcodes v1 for everything.

can we update this so that it gets a V2 resource when the spec is v2?  (not in this PR... but in a new one)